pywren/wait.py

```diff
@@ -14,22 +14,23 @@
 def wait(fs, return_when=ALL_COMPLETED, THREADPOOL_SIZE=64,
          WAIT_DUR_SEC=5):
     """
-    this will eventually provide an optimization for checking if a large
-    number of futures have completed without too much network traffic
-    by exploiting the callset
-
-    From python docs:
-
-    Wait for the Future instances (possibly created by different Executor
-    instances) given by fs to complete. Returns a named 2-tuple of
-    sets. The first set, named "done", contains the futures that completed
-    (finished or were cancelled) before the wait completed. The second
-    set, named "not_done", contains uncompleted futures.
-
+    Wait for the Future instances `fs` to complete. Returns a 2-tuple of
+    lists. The first lists contains the futures that completed
+    (finished or cancelled) before the wait completed. The second
+    contains uncompleted futures.
+
+    :param fs: exception type
+    :param return_when: One of `ALL_COMPLETED`, `ANY_COMPLETED`, `ALWAYS`
+    :param THREADPOOL_SIZE: Number of threads to use. Default 64
+    :param WAIT_DUR_SEC: Time to sleep before checking futures again.
+    :return: list of completed futures, list of uncompleted futures.
```

Comment:
can we also show the signature, e.g., [fs_dones, fs_notdones], then explain what they mean.
If would be nice to have an example here as well. In most cases, we don't save the return objects, as the future objects are updated already.